gensrc/proto/starlake.proto

```diff
@@ -63,8 +63,8 @@ message TxnLogPB {
 message PublishVersionRequest {
     repeated int64 tablet_ids = 1;
     repeated int64 txn_ids = 2;
-    // The version to be published. base version would be version - txn_ids_size()
-    optional int64 version = 3;
+    optional int64 base_version = 3;
```

Comment:
Why did you add `base_version` back?

Reply:
This is no relationship between the base version and the size of txn_ids if one of the transactions is a schema change transaction.

Reply:
Add some comments, PTAL